### PR TITLE
Zoobot/feature/treelist showhide button

### PR DESCRIPTION
**Pull request by zoobot**
Treelist Epic:

Shows and hides the Treelist header on mobile so there's more room to see the tree card or list.

### CODE REVIEW

**Comment by Tijani-zainab** [comment]
@zoobot Lgtm! I'm yet to test the branch out locally. but will do so now. 
Nice work!

File: client/src/tests/ButtonShowHide.test.js [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+    const { getByAltText } = render(
+      <ButtonShowHide showMore={false} setShowMore={mockSetShowMore} />,
+    );
+    const downArrowBtn = getByAltText('show');
+    fireEvent.click(downArrowBtn);
+    expect(mockSetShowMore).toHaveBeenCalledWith(true);
+  });
+
+  it('Handles up arrow (hide) button click', () => {
+    const { getByAltText } = render(
```
Comment (mwpark2014) [inline client/src/tests/ButtonShowHide.test.js:38]: Nice, I need to write these kinds of tests

**Review by mwpark2014 — COMMENTED** [review]
It might be helpful to add a description on the problem these PRs are trying to solve. I'm not sure where this component is going to be used, so I'm not sure if this feedback applies:

I think the one thing I'm concerned about is the expanding library of custom components that might already have implementations in mui. Have you considered using the `Collapse` component (https://codesandbox.io/s/8gv7jz?file=/demo.tsx:1693-1701) or something similar?

**Review by mwpark2014 — APPROVED** [review]
> I did try using Collapse but the custom css I'd need to write over the default seemed like more trouble than it was worth.
I'll a description and take a screenshot.

Okay, I think I can be okay with approving the PR to unblock these changes. I do have some concern that developers will build upon this custom component in the future, which will require updating it to be more extensible and add accessibility, when there might be a reusable component out there that does what we want, but I don't see an easy alternative in MUI and it's prob not worth the time right now

**Pull request merged**